heartbeat/docs/heartbeat-observer-options.asciidoc

```diff
@@ -0,0 +1,7 @@
+[[configuration-observer-options]]
+== Specify Observer and Geo Options
+
+It is often useful to view and query various attributes of the instance heartbeat is running on. This data is attached to events via the <<add-observer-metadata,`add_observer_metadata`>> processor. This processor populates the ECS `observer.*` fields. One field of particular utility is the `observer.geo.name` field, which you can configure via this processor. Use this field to create distinctive geographic regions for your uptime checks. You might label your regions by datacenter name or geographic region, e.g. `virginia-dc-1`, or `us-east-1a`, or `virginia-us`.
```

Comment:
```suggestion
It is often useful to view and query various attributes of the instance Heartbeat is running on. This data is attached to events via the <<add-observer-metadata,`add_observer_metadata`>> processor. This processor populates the ECS `observer.*` fields. One field of particular utility is the `observer.geo.name` field, which you can configure via this processor. Use this field to create distinctive geographic regions for your uptime checks. You might label your regions by datacenter name or geographic region, e.g. `virginia-dc-1`, or `us-east-1a`, or `virginia-us`.
```

Reply:
Hmmm... only change here was heartbeat > Heartbeat. I don't think GitHub handles long line comparisons very well (a good reason to consider adding line breaks).